### ISSUE TITLE
SimpleAudioEngine support

### ISSUE THREAD

**Issue by yonekawa** [rubymotion-community/Joybox#28]
Joybox seems does not contains CocosDenshion.
We can't use SimpleAudioEngine.

```
uninitialized constant AppDelegate::SimpleAudioEngine (NameError)
```

Of course, I can use CocosDenshion by vendor_project.

```
app.vendor_project  'vendor/CocosDenshion', :static
```

However, I want to use it by Joybox only!

**Comment by NSCoder** [comment]
Hey Kenichi!

I will move the Audio Support to the top of the new feature list! I think it will be ready soon, stay tuned!

:D

JK

**Comment by yonekawa** [comment]
Great! Thank you!

**Comment by NSCoder** [comment]
Hey amigo!

Is now fixed in 1.1.0 :)
